### PR TITLE
Merge docs update to stage 

### PR DESCRIPTION
**Pull request by darwin808**
### beta.zesty.io/docs

image

image

image

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -5,3 +5,7 @@ GITHUB_SSO_URL=https://auth.api.dev.zesty.io/github/login
 GOOGLE_SSO_URL=https://auth.api.dev.zesty.io/google/login
 MS_SSO_URL=https://auth.api.dev.zesty.io/azure/login
 TEMPLATE_URL=https://templating.api.zesty.io/
+ALGOLIA_APPID=
```
Comment (ardeay) [inline .env.example:8]: @darwin808 have these values been placed into the dev/stage/prod cloud run config in google cloud?

File: src/pages/docs/[...slug].js [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Docs } from 'views/Docs';
+// import INSTANCE_DATA from '../../views/Docs/instance.data.json';
```
Comment (ardeay) [inline src/pages/docs/[...slug].js]: @giseleblair is the plan to load these remotely from github always?

File: src/pages/docs/[...slug].js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        getPageData(e?.item);
+      }
+    });
+
+    return item;
+  };
+
+  const pageData = getPageData(treeData, mainData);
+
+  console.log(pageData, treeData, 3333333, url, parentUrl);
```
Comment (ardeay) [inline src/pages/docs/[...slug].js]: erroneous  console log

File: src/views/Docs/auth.data.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (ardeay) [inline src/views/Docs/auth.data.json]: Are these postman export files committed to the repo still needed @giseleblair @darwin808

**Review by ardeay — CHANGES_REQUESTED** [review]
I need questions answered in review, also, remove extra console log

File: .env.example (hunk 1 above)
Comment (darwin808) [reply .env.example:8]: yes

File: src/pages/docs/[...slug].js (hunk 2 above)
Comment (giseleblair) [reply src/pages/docs/[...slug].js]: @ardeay yes it is, we have an export integration setup already with the postman collections

File: src/views/Docs/auth.data.json (hunk 4 above)
Comment (giseleblair) [reply src/views/Docs/auth.data.json]: These files can be removed, we will only use the repo data

**Review by ardeay — APPROVED** [review]

**Pull request merged**